src/tests/Microsoft.PowerFx.Connectors.Tests/PowerPlatformConnectorTests.cs

```diff
@@ -1372,5 +1373,145 @@ public async Task ExcelOnlineTest()
 ";
             Assert.Equal(expected, testConnector._log.ToString());
         }
+
+        [Fact]
+        public async Task SQL_ExecuteStoredProc_Scoped()

```

Comment:
>SQL_ExecuteStoredProc_Scoped [](http://example.com/codeflow?start=26&length=28)

This test is evaluating an expression having fixed parameters (in 'globals')

See that the expression has a reduced set of parameters: `SQL.ExecuteProcedureV2(""sp_1"", { p1: 50 })`